test/extensions/filters/network/common/redis/client_impl_test.cc

```diff
@@ -107,6 +242,8 @@ TEST_F(RedisClientImplTest, Basic) {
   Common::Redis::RespValue request1;
   MockPoolCallbacks callbacks1;
   EXPECT_CALL(*encoder_, encode(Ref(request1), _));
+  EXPECT_CALL(*flush_timer_, enabled()).WillOnce(Return(false));
+  ;
```

Comment:
del, same elsewhere.

Reply:
Ah yes, the format checker added those. I'll remove them and ping OSS envoy slack if happens again.